src/resources/js/test/loginForm.test.tsx

```diff
@@ -236,7 +236,7 @@ describe('language selection tests', () => {
     const link = await screen.findByRole('link', { name: 'Benutzer' });
     await user.click(link);
     // we should see the default 'Create New User' even without clicking on the dropdown (Erstelle einen neuen Benutzer in German)
-    await screen.findByText(/Erstelle einen neuen Benutzer/);
+    await screen.findByRole('button', { name: /benutzer erstelle einen neuen benutzer/i });
```

Comment:
change this to 
await screen.findByRole('button', { name: /erstelle einen neuen benutzer/i });

(the code has an extra "benutzer" in it, which is the label of that button rather than the contents -- happens to work but doesn't make sense semantically)

Reply:
fixed.